src/modules/cmdpal/Microsoft.CmdPal.Core.ViewModels/CommandBarViewModel.cs

```diff
@@ -1,15 +1,15 @@
-﻿// Copyright (c) Microsoft Corporation
```

Comment:
I know, you saw "118 files changed" and got spooked. 

Not to worry, almost all the files are this kind of namespace change. You should be able to blow through it ☺️